packages/liveblocks-redux/src/index.ts

```diff
@@ -183,7 +186,7 @@ const internalEnhancer = <T>(options: {
         }
       };
 
-      const store = createStore(newReducer, initialState, enhancer);
```

Comment:
Same goes for the `createStore()` function, which only seems to be taking 2 arguments, not three?

Reply:
Actually, I stand corrected. While the _type definitions_ don't mention it, the documentation does. I'm not going to touch this and revert this change.